### PR TITLE
XAMEETINGS-114:Increase the top spacing for Description and Meetings …

### PR DESCRIPTION
**Pull request by ppantiru**
Added a 12px padding-top for all the labels that might follow the rules of "Description" and "Meeting Notes", so if at any point another will be added, it will have the same 12px padding.

### CODE REVIEW

File: ui/src/main/resources/Meeting/SkinExtension.xml [hunk 1]
```diff
@@ -133,6 +133,9 @@
 
 #durationEdit select{
    width:auto ;
+}
+.col-sm-12.col-xs-12 label {
+  padding-top: 12px;
```
Comment (evalica) [inline ui/src/main/resources/Meeting/SkinExtension.xml]: This is not the solution. We shouldn't add any explicit styling, but just rely on the Bootstrap grid. 
Maybe the classes and the structure is not correct. 

Also there is no difference between after.png and before.png.

File: ui/src/main/resources/Meeting/SkinExtension.xml [hunk 2]
```diff
@@ -133,6 +133,9 @@
 
 #durationEdit select{
    width:auto ;
+}
+.col-sm-12.col-xs-12 label {
```
Comment (evalica) [inline ui/src/main/resources/Meeting/SkinExtension.xml]: Also this kind of specificity for the selector is not wanted. Both are grid classes and they are also targeting different resolutions so there is no semantic reason to use them as selectors.

**Comment by ppantiru** [comment]
I took another approach.
Used $doc.display() instead of $doc.getValue() to generate html instead of wiki syntax so the user cannot break the display so easily(i.e. when there was an empty line). Also a simple <br/> tag ensures the needed space after the map.

**Comment by evalica** [comment]
So apparently in order to fix this issue it was needed the addition of the "break" tag. 
The change from $doc.getValue() to $doc.display() was done as an improvement. Another improvement would have been to move the ``: $doc.display('description')`` outside of the #if in order to remove the code duplication.

**Comment by ppantiru** [comment]
The "break" tag was only part of the solution, the change from $doc.getValue() to $doc.display(), on top of the improvement actually help provide the spacing before "Meeting notes", due to the fact that the "dt" tag is no longer "first-of-type" (because it's no longer interrupted by the newline) so the "margin-top" from the skin kicks in ("dt:first-of-type" is specified in the skin with "margin-top: 0" ).

**Comment by evalica** [comment]
k - thanks for the additional explanations

**Pull request merged**